drivers/SmartThings/zigbee-contact/src/test/test_aqara_contact_sensor.lua

```diff
@@ -86,32 +86,34 @@ test.register_coroutine_test(
   end
 )
 
-test.register_coroutine_test(
-  "closed contact events",
-  function()
-    local attr_report_data = {
-      { IASZone.ID, data_types.Int16.ID, 0x0020}
-    }
-    test.socket.zigbee:__queue_receive({
-      mock_device.id,
-      zigbee_test_utils.build_attribute_report(mock_device, IASZone.ID, attr_report_data, 0x115F)
-    })
-    test.socket.capability:__expect_send( mock_device:generate_test_message("main", capabilities.contactSensor.contact.closed()))
-  end
-)
+--TODO There are no handlers defined for these attr reports
```

Comment:
As alluded to in another comment, the aqara subdriver appears to be expecting some default handlers for the contact sensor events, but no default handlers exist. Similar issues exist for all other subdrivers of the zigbee-contact driver, its just that they dont have tests for contact events. It seems like the subdrivers will all have to specify their own contact event handlers.

Reply:
I see defaults for zigbee contactSensor.  I would expect these messages to be handled.